src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -3815,3 +3815,6 @@
         text: |-
                az restore-point collection wait --resource-group "myResourceGroup" --collection-name "rpcName" --deleted
 """
+
+--public-ip-address
+Name of the public IP address when creating one (default) or referencing an existing one. Can also reference an existing public IP by ID or specify "" for None ('""' in Azure CLI using PowerShell or --% operator). For Azure CLI using powershell core edition 7.3.4, specify '' or "" (--public-ip-address '' or --public-ip-address "")
```

Comment:
Thanks for your contribution. Actually, this is not the correct place to modify the help message for `--public-ip-address`. Please modify the help message here https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/vm/_params.py#L1056.